apps/front/src/components/ui/formInputPassword.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, useState } from 'react'
+import FormInput from '@src/components/ui/formInput'
+import { IconButton, InputAdornment } from '@mui/material'
+import { Visibility, VisibilityOff } from '@mui/icons-material'
+import { FormInputProps } from '@src/types/formInputProps'
+
+const FormInputPassword: FC<FormInputProps> = ({ name, ...otherProps }) => {
+    const [showPassword, setShowPassword] = useState(false)
+
+    const handleClickShowPassword = () => {
+        setShowPassword((prevState) => !prevState)
+    }
+
+    const handleMouseDownPassword = (
+        event: React.MouseEvent<HTMLButtonElement>
+    ) => {
+        event.preventDefault()
```

Comment:
jaki tu przypadek wystąpił że musisz używać `preventDefault`

Reply:
![image](https://github.com/Dawken/Kanban/assets/108353406/854c1dd8-8f98-4ace-8276-566dc2dcea19)

Reply:
Tak było w dokumentacji https://mui.com/material-ui/react-text-field/

Reply:
![image](https://github.com/Dawken/Kanban/assets/108353406/2807abb0-e1f5-4bd0-b98d-d2be767326db)

Reply:
No i co to robi?